scripts/synthetic-network/README.md

```diff
@@ -35,7 +35,23 @@ docker build -t aleph-node:latest -f docker/Dockerfile .
 
 # run e2e-tests
 cd e2e-tests
-# set an ENV variable required by the e2e-test, namely a list of URLs for synthetic-network configuration endpoints
-export SYNTHETIC_URLS="http://localhost:3000/qos,http://localhost:3001/qos,http://localhost:3002/qos,http://localhost:3003/qos,http://localhost:3004/qos"
-cargo test latency
+cargo test --release --no-run --locked
+# copy created binary to e2e-tests/target/release/, built test binary is in the last line of
+# the above command, e.g.
+# cp target/release/deps/aleph_e2e_client-44dc7cbed6112daa target/release/aleph-e2e-client
```

Comment:
Non-blocking: could we automate the synthetic network test setup? E.g. by putting the whole setup in a script?

Reply:
Yes, but not in this PR. Raised task for it.